low-level/files/opt/seagate/sspl/conf/sspl_common_config.yml

```diff
@@ -13,48 +13,39 @@
 # about this software or licensing, please email opensource@seagate.com or
 # cortx-questions@seagate.com.
 
-[SYSTEM_INFORMATION]
```

Comment:
Comment in https://github.com/Seagate/cortx-monitor/pull/281/files#r560360605 applies

Check comments for sspl.conf.LDR_R2 also in same PR

Reply:
addressed comments from  https://github.com/Seagate/cortx-monitor/pull/281 for  sspl.conf.LDR_R2.